testing/internal/e2e/boundary/target.go

```diff
@@ -81,3 +93,15 @@ func AddCredentialSourceToTargetCli(t testing.TB, ctx context.Context, targetId
 	)
 	require.NoError(t, output.Err, string(output.Stderr))
 }
+
+// SetTargetAddressCli uses the cli to assign a network address value to a target
+func SetTargetAddressCli(ctx context.Context, targetId, address string) (string, error) {
+	output := e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"targets", "update", "tcp",
+			"-id", targetId,
+			"-address", address,
+		),
+	)
+	return string(output.Stdout), output.Err
```

Comment:
I see this function doesn't take a `t` at all, but is there a reason not to just check the error in here? I see that we sometimes want to check the exact error, but in those cases, do we even need this helper? We can just use `e2e.RunCommand` directly.

Reply:
I was debating this in my head too, so I will go ahead and remove the helper function and use e2e.RunCommand directly.